lib/bootscale/file_storage.rb

```diff
@@ -1,4 +1,5 @@
 require 'digest/md5'
+require 'active_support/core_ext/file'
```

Comment:
Makes sense. I've copied Sprocket's implementation and updated commit description.